modules/web/client/src/clue/scala/observe/queries/ObsQueriesGQL.scala

```diff
@@ -15,7 +15,7 @@ object ObsQueriesGQL {
     val document = s"""
       query($$site: Site!, $$date: Date!) {
         observationsByWorkflowState(
-          states: [READY, ONGOING],
+          states: [READY, ONGOING, COMPLETED],
```

Comment:
Would this return a lot of completed observations?

Reply:
Only the ones that are active and are associated to the current site.